package.json

```diff
@@ -120,9 +120,9 @@
     "react-docgen-typescript-loader": "3.7.2",
     "react-dom": "16.13.1",
     "react-test-renderer": "16.13.1",
+    "terser-webpack-plugin": "^4.2.3",
```

Comment:
I had to replace the uglify minifier because it is deprecated and couldn't parse the new dependencies.